src/main/java/pl/vm/javaguild/designpatterns/pattern/creational/abstractfactory/model/enums/Producer.java

```diff
@@ -0,0 +1,5 @@
+package pl.vm.javaguild.designpatterns.pattern.creational.abstractfactory.model.enums;
+
+public enum Producer {
```

Comment:
Proszę o dodanie krótkiego Javadoca.

Dodatkowo sugeruję, aby każda z wartości Enuma była w osobne linii - poprawia to czytelność i ułatwia rozszerzalnośc.